docs/New_Github_Action_Workflow.md

```diff
@@ -0,0 +1,53 @@
+# CI Pipeline Naming Convention
+
+To improve navigation and consistency within our GitHub Actions workflows, **all new CI pipelines that run on pull requests must have names beginning with `(CI)`**. This naming convention is designed to limit the workflows shown in the GitHub Actions web UI to those from the main branch at the top, reducing clutter from workflows in other branches.
+
+## Rationale
+
+- **Easy Identification:**  
+  Prefixing CI pipelines with `(CI)` makes it straightforward to identify and filter Continuous Integration workflows, especially those that run on pull requests.
+
+- **Enhanced Navigation in the Web UI:**  
+  By standardizing the naming of CI workflows on the main branch, these pipelines will be shown on top of the pipelines in the GitHub Actions sidebar, reducing the noise from workflows in feature branches.
+
+- **Consistent Naming:**  
+  A uniform naming convention helps maintain clarity and consistency across our CI/CD pipelines.
+
+## Guidelines
+
+1. **Naming Format:**  
+   All new CI workflows that run on pull requests must start with the prefix `(CI)`.  
+   - **Example:** `(CI) Build & Test`, `(CI) Lint & Validate`, `(CI) Integration Tests`
+
+2. **Workflow File Location:**  
+   Place your CI pipeline YAML files in the `.github/workflows` directory of the repository.
+
+3. **Review Process:**  
+   During code reviews, verify that any new CI pipeline adheres to the naming convention before merging. This will ensure that only workflows from the main branch (with the `(CI)` prefix) appear at the top of the GitHub Actions sidebar.
```

Comment:
Please add more details here. When you discussed it with me, you said each PR must be approved twice? Please explain the process in more detail.

Reply:
Update the guidance, this is no hard enforcement. If someone forget to do it, any of us can address it. The goal is to make everyone’s life easier by keeping our CI workflows organized.